tests/State/ScopeTest.php

```diff
@@ -29,57 +29,68 @@ public function testSetTag(): void
         $this->assertSame(['foo' => 'bar', 'bar' => 'baz'], $event->getTagsContext()->toArray());
     }
 
-    public function setTags(): void
```

Comment:
`setTags` was probably pasted as reference when writing the tests and accidentally committed. Wasn't being used, so I removed it.